### PR TITLE
[Do not merge] Try enable code coverage.

### PR DESCRIPTION
**Pull request by katmsft**

### CODE REVIEW

File: sdk/storage/ci.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ stages:
       ServiceDirectory: storage
       # TODO: Change to azure-storage once we have an strategy to run livetests or use test recordings
       CtestRegex: azure-core
-      Coverage: disabled
+      Coverage: enabled
```
Comment (vhvb1989) [inline sdk/storage/ci.yml:40]: This needs to stay disabled. It is for CI tests only.
CodeCoverage will be ON for LiveTests by default.  I have enabled it here: https://github.com/Azure/azure-sdk-for-cpp/pull/939

**Review by vhvb1989 — COMMENTED** [review]
The `Coverage` setting from ci.yml work only if there are tests ran in the client - CI pipeline.
Storage does not run any unit tests on that pipeline. All tests are directly run on LiveMode (where code coverage must be ON all the times).
In the future, once we have the record&playback tests, you can enable `Coverage` to add more coverage from the client ci pipeline

File: sdk/storage/ci.yml (hunk 1 above)
Comment (katmsft) [reply sdk/storage/ci.yml]: Closing the PR.

**Pull request closed without merging**